README.md

```diff
@@ -1,2 +1,195 @@
-# dogecoin.com
+# Dogecoin.com
 Temporary repo for dogecoin.com redesign
+
+## How to get up and running:
+I'm working off of a redesign branch and reconfigured gh-pages.yml to exclusively build and deploy this branch name. After you clone in order to make this work for github pages please checkout a new branch called redesign in your respective fork by doing:
+
+`git checkout -b redesign main`
```

Comment:
should the readme also be revised since these changes will be merged to the `main` branch?

Reply:
absolutely, this is was more or less to help newcomers get started or at least know the current state of the redesign.